config/globalSetup.ts

```diff
@@ -0,0 +1,6 @@
+require('ts-node/register');
```

Comment:
There is a lot of vague references to what this does in the `ts-node` README... can I get a quick one liner on what this is helping us with?

My guess is it is to setup the commonJS stuff for node, but this can be added to the `tsconfig` if I understand the setup correctly.

Reply:
I was not aware of the `ts-node` config option, let me give that a try.

Reply:
For posterity, [ts-node docs](https://github.com/TypeStrong/ts-node#via-tsconfigjson-recommended)

Reply:
@andrewballantyne This allows `ts-node` to hook into the Node `require` machinery, so when a module is loaded, it can transform it via TS compiler.